docs/user-guide/continuous-aggregation/overview.md

```diff
@@ -133,6 +133,8 @@ Here is the explanation of the columns in the `ngx_statistics` table:
 - `time_window`: The time window of the aggregation.
 - `update_at`: The time when the aggregation is updated.
 
+NOTE: if you don't manually create sink table, the Flow engine will automatically create it for you based on the query(i.e. using columns in `GROUP BY` as primary tags and time index), however, sometimes you may want to create the sink table manually to have more control over the schema.
```

Comment:
Please do not use NOTE to introduce this feature.
You can modify the exist content to show the automatically table creation. for example, remove the `CREATE TABLE `ngx_statistics` statement, add the `desc ngx_statistics` after `SELECT * FROM ngx_statistics` and explain the feature about automatically table creation and explain each column of the schema.